tests/test_make.py

```diff
@@ -546,6 +546,18 @@ class C(object):
 
         assert "mutually exclusive" in ei.value.args[0]
 
+    def test_sugar_callable(self):
+        """
+        Factory has to be a callable to prevent people from passing Factory
+        into it.
+        """
+        with pytest.raises(ValueError) as ei:
```

Comment:
`with pytest.raises(ValueError, match="must be a callable"): ...` might make this more pretty

Reply:
ohhh…shiny new pytest feature or am I just out of touch? :)

Reply:
https://github.com/pytest-dev/pytest/pull/2227 was merged a year ago, but its a detail easy to miss